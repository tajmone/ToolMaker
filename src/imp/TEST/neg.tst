%%BEGIN(comimIntToHex(num))
%%(num)
%%END(comimIntToHex)

%%INCLUDE(comimIntToHex(100))
%%INCLUDE(comimIntToHex(-100))
%%SET f(-5)
%%INCLUDE(comimIntToHex(%%(-f)))
%%EINCLUDE(comimIntToHex(3--5*-3))
%%INCLUDE(comimIntToHex(--7))
