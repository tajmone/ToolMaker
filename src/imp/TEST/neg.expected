########## neg ##########
%%BEGIN(comimIntToHex(num))
%%(num)
%%END(comimIntToHex)

%%INCLUDE(comimIntToHex(100))
%%INCLUDE(comimIntToHex(-100))
%%SET f(-5)
%%INCLUDE(comimIntToHex(%%(-f)))
%%EINCLUDE(comimIntToHex(3--5*-3))
%%INCLUDE(comimIntToHex(--7))
########## ../imp neg.tst >neg.tst.tmp; cat neg.tst.tmp; rm neg.tst.tmp ##########

100
-100
5
-12
7
